.github/workflows/molecule_tests.yml

```diff
@@ -74,6 +62,14 @@ jobs:
           - pulibrary.apache_maven
           - pulibrary.pulmap
           - pulibrary.memcached
+  test-molecule3:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        role:
+          - figgy
```

Comment:
I would expect redis, ruby, subversion, vips, nodejs, bind9, freetds, passenger, &  rails-app all to be here...  Was there a reason for no moving them?

Reply:
done.